src/components/tutor-schedule/TutorSchedule.constants.ts

```diff
@@ -0,0 +1,25 @@
+export const items = [
```

Comment:
This file is redundant. For tast purpose this mock can be in one file with component